Tests/Linq/Linq/MathFunctionTests.cs

```diff
@@ -199,6 +199,15 @@ from t in from p in    Types select Math.Floor(Math.Pow((double)p.MoneyValue, 3)
 					from t in from p in db.Types select Math.Floor(Math.Pow((double)p.MoneyValue, 3)) where t != 0 select t);
 		}
 
+		[Test]
+		public void PowDecimal([DataSources(ProviderName.SQLiteMS)] string context)
```

Comment:
Changes to disable ASE testing. Not worth fixing for now
```suggestion
		// Sybase: https://stackoverflow.com/questions/25281843
		[Test]
		public void PowDecimal([DataSources(ProviderName.SQLiteMS, TestProvName.AllSybase)] string context)
```